cartographer/sensor/point_cloud.h

```diff
@@ -28,8 +28,9 @@
 namespace cartographer {
 namespace sensor {
 
-typedef std::vector<Eigen::Vector3f> PointCloud;
-typedef std::vector<Eigen::Vector2f> PointCloud2D;
+
+typedef std::vector<Eigen::Vector3f, Eigen::aligned_allocator<Eigen::Vector3f>> PointCloud;
```

Comment:
This and most of the other changes are unneeded. Here for example, `Eigen::Vector3f` is too small to be vectorizable, see https://eigen.tuxfamily.org/dox/group__TopicFixedSizeVectorizable.html.